docs/src/main/asciidoc/kubernetes-client.adoc

```diff
@@ -142,7 +142,7 @@ Note that to take advantage of these features, the `quarkus-test-kubernetes-clie
 == Note on implementing the Watcher interface
 
 Due to the restrictions imposed by GraalVM, extra care needs to be taken when implementing a `io.fabric8.kubernetes.client.Watcher` if the application is intended to work in native mode.
-Essentially every `Watcher` implementation needs to specify the Kubernetes model class that it handles via the `Watcher`'s generic type at class definition time.
+Essentially every `Watcher` implementation needs to specify the Kubernetes model class that it handles via the `Watcher`&#8217;s generic type at class definition time.
```

Comment:
IIRC, in this case, you should use:
```
``Watcher``'s
```
with 2 backticks.

Reply:
I thought i had tried that, guess not. Oh well, I've changed the doc and created new pull request.